<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateuser</class>
 <widget class="QWidget" name="updateuser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;用户名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_value">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请设置账号</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>571</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>400</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_value">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请设置密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add</sender>
   <signal>clicked()</signal>
   <receiver>updateuser</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del</sender>
   <signal>clicked()</signal>
   <receiver>updateuser</receiver>
   <slot>del()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back</sender>
   <signal>clicked()</signal>
   <receiver>updateuser</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>update</sender>
   <signal>clicked()</signal>
   <receiver>updateuser</receiver>
   <slot>up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>del()</slot>
  <slot>back()</slot>
  <slot>up()</slot>
 </slots>
</ui>
